generator.bzl

```diff
@@ -1,3 +1,5 @@
+# Changed 2020 by Zenseact AB
```

Comment:
This is to comply with the Apache 2.0 license §4b, "You must cause any modified files to carry prominent notices stating that You changed the files."

If the file had the standard boilerplate header, http://www.apache.org/licenses/LICENSE-2.0#apply, I think the normal way to comply with §4b would be to just add another line with a copyright statement, but I don't feel I can just insert a Zenseact copyright statement if there wasn't any to begin with. Do you want this type of notice, and how would you like it to look?